server/test/unit/server/managers/consumer/test_agent.py

```diff
@@ -231,6 +231,8 @@ def test_install_content(self, *mocks):
         mock_context.assert_called_with(consumer, task_id=task_id, consumer_id=consumer['id'])
         mock_profiler.install_units.assert_called_with(consumer, [unit], options, {}, ANY)
         mock_agent.install.assert_called_with(mock_context.return_value, [unit], options)
+        mock_factory.consumer_history_manager().record_event.assert_called_with(
```

Comment:
I think you need a similar mock assertion for the `history_manager.record_event(consumer_id, 'content_unit_uninstalled', {'units': units})` in agent.py